samples/sample_hevc_fei/src/pipeline_hevc_fei.cpp

```diff
@@ -472,6 +471,7 @@ MfxVideoParamsWrapper GetEncodeParams(const sInputParams& user_pars, const mfxFr
     pars.mfx.NumRefFrame  = user_pars.nNumRef;
     pars.mfx.NumSlice     = user_pars.nNumSlices;
     pars.mfx.EncodedOrder = user_pars.bEncodedOrder;
+    pars.mfx.TargetKbps   = user_pars.TargetKbps;
```

Comment:
It looks really strange that RateControlMethod is still CQP when TargetKbps can be non-zero

Reply:
TargetKbps is set to 0 if ExtBRC is not specified.

Reply:
It's no obvious in this code

Reply:
agree. not sure how to do it more clear and to minimize the number of different checking